rest_framework/compat.py

```diff
@@ -33,6 +33,14 @@ def unicode_to_repr(value):
     return value
 
 
+def total_seconds(timedelta):
+    # TimeDelta.total_seconds() is only available in Python 2.7
+    if hasattr(timedelta, 'total_seconds'):
+        return timedelta.total_seconds()
+    else:
+        return (timedelta.microseconds + (timedelta.seconds + timedelta.days * 24 * 3600) * 10 ** 6) / 10 ** 6
```

Comment:
Ah forget the previous question - `total_seconds()` is float, so my above inline comment is correct.
And yes, I see the python docs writing the expression a different way around, but the above style is far clearer to my eyes.